packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -188,6 +191,11 @@ void main() {
     "description": "The conventional newborn programmer greeting"
   },
 
+  "helloNewlineWorld": "Hello \n World",
+  "@helloNewlineWorld": {
+    "description": "A message with a backslash followed by an n"
```

Comment:
This is a weird area of course. You could mention that the JSON decoder will convert the backslash-n to a newline.

Reply:
I don't quite follow. Do you mean explaining in the description that the JSON decode converts a backslash-n into a newline character in the generated Dart string?

Reply:
Done

Reply:
Yes. It's a backslash-n here but JSON.decode() will convert that into a newline. I realize that this all seems a bit pedantic; on the other hand, it's been a source of some confusion too.